great-idea/css/index.css

```diff
@@ -63,4 +63,174 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container{
+	width:1000px;
+	margin: 0 auto;
+}
+
+.top {
+	display: flex;
+	justify-content: space-between;
+	height: 50px;
+}
+
+.main-nav {
+	display: flex;
+	justify-content: space-evenly;
+	width: 800px;
+	margin-top:10px;
+}
+
+.main-nav a {
+	color:gray;
+	text-decoration: none;
+}
+
+.demand {
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+}
+.start {
+	border: 1px solid black;
+	align-self: flex-start;
+	padding: 10px;
+}
+
+.demand1 {
+	display: flex;
+	width: 1000px;
+	justify-content: space-evenly;
+
+}
+.demand1 h1 {
+	align-self: center;
+	font-size: 60px;
+	text-align: center;
+}
+.demand1 img {
+	margin-top:10px
+}
+
+.first-section {
+	display: flex;
+	justify-content: space-between;
+}
+.para {
+	width: 450px;
+}
+.para2 {
+	width: 300px;
+}
+
+.middle-img {
+	width: 1000px;
+	height: auto;
+}
+.middle-image{
+	height: 250px;
+	display: flex;
+	align-items: center;
+}
```

Comment:
I see that you are push elements away with using the height property. It gets the job done but it isnt the correct/designated property we want to use to push other elements away. I want you to figure out which property can you use instead of height . Same applies with other elements that u set to use height to achieve the styling.